src/boot/efi/secure-boot.c

```diff
@@ -120,7 +120,7 @@ EFI_STATUS secure_boot_enroll_at(EFI_FILE *root_dir, const char16_t *path, bool
         if (!is_safe)
                 printf("\n");
```

Comment:
If you touch this, I'd like to move this newline output to the `!is_safe` if branch above. It is currently difficult to follow.